package.json

```diff
@@ -54,6 +54,6 @@
 		"pa11y": "./bin/pa11y"
 	},
 	"scripts": {
-		"test": "make ci"
+		"test": "snyk test && make ci"
```

Comment:
Actually, all of this has been added by the snyk CLI automatically. Seems that it's not smart enough to realise that we're running `make test` in Travis instead of `npm test` (which probably we should be using anyway - it seems bad practice having to specify the same command in two different places).

Let me study this it a bit further and we can discuss it in Slack later.